docs/fastdds/dds_layer/subscriber/dataReader/readingData.rst

```diff
@@ -0,0 +1,35 @@
+.. _dds_layer_subscriber_accessreceived:
+
+Accessing received data
```

Comment:
I think this section should include the two modes of data retrieving:
* Using listener and callbacks
* Waiting on the thread until a sample is received. This can be done with `DataReader::wait_for_unread_message(const fastrtps::Duration_t& timeout)`